it-tests/settings/CatalogURLSettings.test.ts

```diff
@@ -53,46 +53,15 @@ describe('User Settings', function () {
 				galleryButton: By.xpath(`//button[@id='toggle-layout-button-Gallery']`),
 				closeButton: By.xpath(`//button[@data-ouia-component-id='CatalogModal-ModalBoxCloseButton']`),
 			},
+			catalogProviderSelector: {
+				dropdown: By.xpath(`//button[@data-testid='provider-filter-toggle']`),
+				menu: By.id('providers-select-menu'),
+				menuItem: By.className('pf-v6-c-menu__list-item'),
```

Comment:
We can move forward with this for now, but this class name will change when upgrading patternfly :( 

Can you please let me know which element you need to access so we can provide a `data-testid` or maybe a more static accessor?